Cargo.toml

```diff
@@ -63,14 +68,18 @@ tar = "0.4"
 time = { version = "0.3", features = ["serde-well-known"] }
 thiserror = "1"
 tokio = { version = "1", default-features = false, features = ["full"] }
-tokio-stream = { version = "0.1", default-features = false, features = ["fs", "sync"] }
+tokio-stream = { version = "0.1", default-features = false, features = [
+    "fs",
+    "sync",
+] }
 tokio-tungstenite = "0.23"
 toml = "0.8"
 tower-http = { version = "0.5.1", features = ["fs", "trace", "set-header"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 which = "6"
 zip = "2"
+homedir = "0.3.3"
```

Comment:
This was the only real change on this page, the rest was `rustfmt`.

Reply:
Could you sort that alphabetically.

Reply:
Done